source/developer/fx-guidelines.rst

```diff
@@ -391,18 +396,25 @@ Input: Hints
     Incorrect:
         When a new message is received, flash the taskbar icon.
 
+    Ensure plural interpretations work when applicable and test help text with machine translation to reduce non-obvious, English-specific jardon:
     
+    Correct:
+        There are a few configuration settings you can adjust when setting up your instance of Mattermost.
```

Comment:
As a general rule, you should avoid starting a sentence with "There are..." because it moves the subject of the sentence way out to the middle or the end and sometimes makes it non-obvious what the subject of the sentence actually is. In this case, the subject is either "you" or "your instance of Mattermost". You can rewrite it in two ways:

1. You can adjust a few configuration settings when setting up your instance of Mattermost.
1. Your Mattermost instance has a few configuration settings that can be adjusted when setting it up.

Both put the subject at the beginning where it's easier to understand. The first example is better because it uses active voice and is in the imperative mood (present tense, second person)